content/blog/2020/03/2020-03-02-pipeline-authoring-sig-update.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Pipeline-Authoring SIG Update"
+tags:
+- Pipeline-Authoring
+- SIG
+- community
+- opensource
+author: markyjackson-taulia
+---
+
+
+==== What is the Pipeline-Authoring Special Interest Group
+
+This special interest group aims to improve and curate the experience of authoring Jenkins Pipelines. This includes the
+syntax of `Jenkinsfile`s and shared libraries, code sharing and reuse, testing of Pipelines and shared libraries, IDE
+integration and other development tools, documentation, best practices, and examples.
+
+=== What Are The Focus Areas of the Pipeline-Authoring Special Interest Group
+
+* Syntax - How `Jenkinsfile`s and shared libraries are written.
+* Code sharing and reuse - Shared libraries and future improvements.
+* Testing - Unit and functional testing of `Jenkinsfile`s and shared libraries.
+* IDE integration, editors, and other development tools - IDE plugins, visual editors, etc.
+* Documentation - Reference documentation, tutorials, and more.
+* Best practices - Defining, maintaining, and evangelizing best practices in Jenkins Pipeline.
+* Examples - Real-world `Jenkinsfile`s and shared libraries demonstrating how to utilize various features of Pipeline,
+as well as basic or starter `Jenkinsfile`s for common patterns that can be used as jumping-off points by new users.
+
+
+=== What Have We Been Up To
+evangelizing
```

Comment:
I believe that asciidoc wants a blank line between heading text and paragraph text.  Even if it does not, this word seems misplaced.
```suggestion

```